pom.xml

```diff
@@ -87,7 +87,7 @@
                         <phase>process-test-classes</phase>
                         <configuration>
                             <target>
-                                <chmod file="target/test-classes/chromedriver/windows/chromedriver.exe" perm="755" />
+                                <chmod file="target/test-classes/chromedriver/windows/chromedriver" perm="755" />
```

Comment:
This should be having an extenson .exe

Reply:
@kdaud did you respond to this comment?

Reply:
> This should be having an extension .exe

Once the driver is executable which certainly should be made an executable one, then we do not need to have an extension of `.exe` in our configurations

Here are the configurations fo linux and mac os

Mac os [chrome ](https://github.com/openmrs/openmrs-contrib-qaframework/blob/b60679a483fd6e18535b116070f2f2018434f35f/pom.xml#L64)and [firefox
](https://github.com/openmrs/openmrs-contrib-qaframework/blob/b60679a483fd6e18535b116070f2f2018434f35f/pom.xml#L103)

Linux os [chrome](https://github.com/openmrs/openmrs-contrib-qaframework/blob/b60679a483fd6e18535b116070f2f2018434f35f/pom.xml#L77) and [firefox](https://github.com/openmrs/openmrs-contrib-qaframework/blob/b60679a483fd6e18535b116070f2f2018434f35f/pom.xml#L116)